src/components/EditSections/EditServicePoints/EditServicePoints.test.js

```diff
@@ -0,0 +1,109 @@
+
+import React from 'react';
+import okapiCurrentUser from 'fixtures/okapiCurrentUser';
+import { screen, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Form } from 'react-final-form';
+
+import renderWithRouter from 'helpers/renderWithRouter';
+
+import EditServicePoints from './EditServicePoints';
+
+
+import '__mock__/stripesCore.mock';
+import '__mock__/stripesSmartComponent.mock';
+
+jest.unmock('@folio/stripes/components');
+
+
+const onSubmit = jest.fn();
+
+
+const arrayMutators = {
+  concat: jest.fn(),
+  move: jest.fn(),
+  pop: jest.fn(),
+  push: jest.fn(),
+  remove: jest.fn(),
+  removeBatch: jest.fn(),
+  shift: jest.fn(),
+  swap: jest.fn(),
+  unshift: jest.fn(),
+  update: jest.fn()
+};
+
+
+const renderEditServicePoints = (props) => {
+  const component = () => (
+    <>
+      <EditServicePoints {...props} />
+    </>
+  );
+  renderWithRouter(
+    <Form
+      id="form-user"
+      mutators={{
+        ...arrayMutators
+      }}
+      initialValues={{
+        servicePoints: okapiCurrentUser.servicePoints,
+      }}
+      onSubmit={onSubmit}
+      render={component}
+    />
+  );
+};
+
+
+
+const onToggleMock = jest.fn();
+const onChangeMock = jest.fn();
+
+const propData = {
+  accordionId: 'servicePoints',
+  expanded: true,
+  initialValues: {
+    servicePoints: okapiCurrentUser.servicePoints,
+  },
+  onToggle: onToggleMock,
+  intl: { formatMessage : jest.fn() },
+  formData: {
+    servicePoints: okapiCurrentUser.servicePoints,
+  },
+  form: {
+    change: onChangeMock,
+  },
+  navigationCheck: true
+};
+
+
+describe('Edit Service Points Component', () => {
+  describe('With Service Data', () => {
+    beforeEach(() => {
+      renderEditServicePoints(propData);
+    });
+    afterEach(cleanup);
+    it('Check if modal Renders', () => {
+      expect(screen.getAllByText('Online')[0]).toBeInTheDocument();
+      expect(screen.getAllByText('Circ Desk 2')[0]).toBeInTheDocument();
+      expect(screen.getAllByText('Circ Desk 1')[0]).toBeInTheDocument();
+    });
+    it('Check if add  service point modal works', () => {
+      userEvent.click(document.querySelector('[id=add-service-point-btn]'));
+      expect(screen.getByText('ui-users.sp.addServicePoints')).toBeInTheDocument();
+      userEvent.click(screen.getByText('stripes-core.button.cancel'));
```

Comment:
Can this line be removed? I don't think there's any value in a `userEvent` after we're done making assertions.